### PR TITLE
UX: Less specific styling for Auth logins

### PR DESCRIPTION
**Pull request by jordanvidrine**
In order to better adhere to branding guidelines this PR changes the text of some of the auth sign in / log in buttons.

## After

**Sign up**
image

**Login in**
image

## Before
**Sign up**
image

**Log in**
image

### CODE REVIEW

**Review by pmusaraj — COMMENTED** [review]
Sadly, it looks like this still doesn't conform to guidelines. For example, Facebook says this: 

> DON’T place the call to action copy (example: Continue with Facebook) outside of the login button.

What makes matters worse is that different providers us different language. For example, Facebook wants "Login with Facebook" or "Continue with Facebook", Google prefers "Sign in with Google". 

References 
https://developers.facebook.com/docs/facebook-login/userexperience/ 
https://developers.google.com/identity/branding-guidelines

**Comment by jordanvidrine** [comment]
**New FB button**
image

https://developers.facebook.com/docs/facebook-login/userexperience/#:~:text=FACEBOOK%20BLUE%20COLOR%20VALUES

**New Google Button**
image

https://developers.google.com/identity/branding-guidelines#color

**New twitter button**
image

https://developer.twitter.com/en/docs/authentication/guides/log-in-with-twitter#:~:text=Log%20in%20with%20Twitter%20Resources

File: app/assets/stylesheets/desktop/login.scss [hunk 1]
```diff
@@ -14,13 +14,16 @@
       color: rgba(var(--primary-rgb), 0.85);
     }
   }
+  h2.login-with {
```
Comment (pmusaraj) [inline app/assets/stylesheets/desktop/login.scss:17]: This is no longer useful.

File: app/assets/stylesheets/desktop/login.scss [hunk 2]
```diff
@@ -14,13 +14,16 @@
...
+    color: var(--secondary);
+  }
   #login-buttons:not(.hidden) {
     display: flex;
     flex: 0 1 auto;
     flex-direction: column;
     align-items: stretch;
     min-height: 75px;
-    min-width: 160px;
+    min-width: 200px;
```
Comment (pmusaraj) [inline app/assets/stylesheets/desktop/login.scss:26]: Does this cause issues on small-ish window sizes?

**Review by pmusaraj — COMMENTED** [review]
Can you update screenshots in OP? 

We also need to update some plugins that hook into this UI (Apple, Microsoft, LinkedIn, VKontakte, probably more).

File: app/assets/stylesheets/desktop/login.scss (hunk 2 above)
Comment (jordanvidrine) [reply app/assets/stylesheets/desktop/login.scss:26]: How small are we thinking? It currently looks fine up until about 520px, then a horizontal scroll kicks in on the modal. 

I feel this is fine for now, as we are eventually moving to full page right?

**Review by pmusaraj — APPROVED** [review]

**Pull request merged**